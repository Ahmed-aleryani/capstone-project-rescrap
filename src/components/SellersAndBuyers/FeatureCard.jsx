import React from 'react';
import Title from '../Title/Title';
import Description from '../Description/Description';

const FeatureCard = ({imgPath, imgDesc, featHeader, featDesc}) => {
    return ( <div className="w-64 text-center">
        <img className="h-24 w-24 inline-block bg-green-400" src={imgPath} alt={imgDesc}/>
        <Title classes=" my-3" text={featHeader}/>
        <Description classes="" text={featDesc}/>
    </div> );
}
 
export default FeatureCard;