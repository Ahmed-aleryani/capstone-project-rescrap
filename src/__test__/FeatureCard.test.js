import React from 'react';
import renderer from 'react-test-renderer';
import FeatureCard from './../components/SellersAndBuyers/FeatureCard';


it("matches the snapshot",()=>{
    const tree = renderer.create(<FeatureCard/>).toJSON();
    expect(tree).toMatchSnapshot();
})