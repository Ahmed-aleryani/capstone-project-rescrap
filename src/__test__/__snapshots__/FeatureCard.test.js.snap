// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div
  className="w-64 sm:pb-16 text-center undefined"
>
  <img
    className="h-24 w-24 inline-block bg-green-400"
  />
  <h2
    className="text-2xl thin my-3"
  />
  <p
    className="text-base font-normal "
  />
</div>
`;
